app/models/hosts_storage.rb

```diff
@@ -1,2 +1,4 @@
 class HostsStorage < ActiveRecord::Base
```

Comment:
Hmm I'm getting the same error with that while using save_inventory_multi and update_attributes with the primary_key set explicitly to nil.  We were trying to set a composite primary key with host_id and storage_id but that needed an extra gem for rails to be able to do that, not great but another option.